.github/scripts/stop_runner_service.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set +e
+set -x
+
+# Get the service name
+RUNNER_SERVICE=$(cat "${RUNNER_WORKSPACE}/../../.service")
+echo "GitHub self-hosted runner service: ${RUNNER_SERVICE}"
+
+if [[ -n "${RUNNER_SERVICE}" ]]; then
```

Comment:
This script will not do anything if service have crashed ? If service is not available we should probably try to trigger service shutdown